src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -661,11 +666,16 @@ def _set_no_abort(self):
 
     def _reset_cache(self):
         """Reset cached items"""
-        self._mesh_rep._reset_cache()
+        if self._mesh_rep is not None:
+            self._mesh_rep._reset_cache()
         self._geometry._reset_cache()
 
     @property
     def _mesh(self):
+        if self._mesh_rep is None:
```

Comment:
Ignore coverage because we can't test this.
```suggestion
        if self._mesh_rep is None:  # pragma: no cover
```